web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1567,6 +1567,16 @@ export const formatters: Formatters = {
     format: ({ name, source, user }) =>
       `Okta assignment [${name}], source [${source}], user [${user}] cleanup has failed`,
   },
+  [eventCodes.OKTA_USER_SYNC]: {
+    type: 'okta.user.sync',
+    desc: 'Okta user synchronization completed',
```

Comment:
Should we say something like "Okta sync added X and removed Y users"

It would make it easier to see what happened at a glance without clicking into the event details.